### DISCUSSION TITLE
Am getting Not Logged In errror with Netflix

### DISCUSSION THREAD

**Discussion by eskwire** [trakt-tools/universal-trakt-scrobbler#401] [Q&A]
As the tittle says I can sync from Netflix cus I get this error. https://i.imgur.com/NiCSOVV.png  And am loogging on netflix I deleted the info log back in and try another browser but same problem chrome and firefox.

**Comment by TDanks2000** [comment]
for the time being, build from my pr https://github.com/trakt-tools/universal-trakt-scrobbler/tree/40cea2baf9456350421a6fef227bc3a9fe4a3641

and add the chrome or firefox folder to your chosen browser

**Reply by eskwire** [reply]
Ok will try it to see. thanks.

**Comment by djfluffkins** [comment]
This issue seems to be happening again and I'm on the version post the commit you had. Did it somehow change again?

**Reply by eskwire** [reply]
No that did not solve the problem. I resources to do it on the app trakt since the plugin stop work. There are other options out there but they dont play nice with seedbox for me.
